server/handler.go

```diff
@@ -242,6 +251,24 @@ func (h *Handler) PauseOrResumeScheduler(name string, t int64) error {
 	return err
 }
 
+// PauseOrResumeChecker pauses checker for delay seconds or resume checker
+// t == 0 : resume checker.
+// t > 0 : checker delays t seconds.
```

Comment:
What if t < 0?

Reply:
t < 0 will behave the same as t == 0, i.e. resume the checker, if we do not consider `int64` overflow. But passing a argument t < 0 should not encouraged. Should this be documented?

Reply:
Yes, or return a request error?

Reply:
Fixed. A negative delay now gives a 400 status.